schema.sql

```diff
@@ -0,0 +1,21 @@
+
+
+  CREATE TABLE author(
+    id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    first_name  varchar,
+    last_name varchar,
+  )
+
+CREATE TABLE game(
+  id INT GENERATED ALWAYS IDENTITY PRIMARY KEY,
+  gener INT,
+  author INT,
+  label INT,
+  publish_date DATE,
+  archived BOOLEAN,
+  multiplayer INT,
+  last_played_at DATE,
+  CONSTRAINT gener_fk FOREIGN KEY(gener) REFERENCES gener(id),
+  CONSTRAINT author_fk FOREIGN KEY(author) REFERENCES author(id),
+  CONSTRAINT label_fk FOREIGN KEY(label) REFERENCES label(id)
+  );
```

Comment:
It would be a great idea to create the ITEM table and reference it within the GAME table.

Reply:
Actually. We don't need to create the Item table. Hence, there is a one-to-one relationship between `items` (book, music, game) and other tables (author, genre, label, source) we just need a `foreign key` to the one side. But let's discuss that when you guys are available.